shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -894,16 +885,19 @@ - (void)setTextInputClient:(int)client withConfiguration:(NSDictionary*)configur
     _activeView = isSecureTextEntry ? _nonAutofillSecureInputView : _nonAutofillInputView;
     [FlutterTextInputPlugin setupInputView:_activeView withConfiguration:configuration];
 
-    if (![_activeView isDescendantOfView:_inputHider]) {
-      [_inputHider addSubview:_activeView];
+    if (!_activeView.window) {
```

Comment:
`if (_activeView.window != keyWindow)` would be a bit safer.

Reply:
ah right, thanks